content/en/service_management/service_level_objectives/_index.md

```diff
@@ -288,7 +288,30 @@ To view, edit, and delete existing status corrections, click on the **Correction
 
 The SLO Calendar View is available on the [SLO status page][2]. On the top right corner, switch from the "Primary" view to the "Weekly" or "Monthly" view to see 12 months of historical SLO status data. The Calendar View is supported for Metric-based SLOs and Time Slice SLOs.
 
-{{< img src="service_management/service_level_objectives/calendar-view-slo.png" alt="SLO calendar view" >}}
+{{< img src="service_management/service_level_objectives/slo-calendar-view.png" alt="SLO calendar view" >}}
+
+## SLO CSV Export (Private Beta)
+
+{{< callout url="https://forms.gle/GQkcHDqaL5qWMss38" btn_hidden="false" header="Try Out the SLO CSV Export Feature">}}
+The CSV Export feature is in Private Beta. Complete the form to request access.
+{{< /callout >}}
+
+The SLO CSV Export feature is available on the [SLO status page][2] once you switch to the "Weekly" or "Monthly" Calendar View. In these views, can access the new “Export to CSV” option where you can download a CSV of your historical SLO data:
+
+- The SLOs that are exported are based on your search query.
+- The Calendar View is supported for Metric-based and Time Slice SLOs. If you export any Monitor-based SLOs, only the SLO id and name will be in the CSV (not the SLO’s status history data).
+- There is a limit of 1000 SLOs per export.
+
+{{< img src="service_management/service_level_objectives/slo-csv-export.png" alt="SLO calendar view" >}}
+
+The following time windows are available for the CSV export:
+
+- **Weekly:** The SLO statuses are based on calendar-aligned weeks (Sunday 12am - Saturday 11:59pm)
```

Comment:
We might want to add that the interval alignment is based on the user timezone set for their datadog user. They can set it explicitly in the API